### DISCUSSION TITLE
Tell me how you can generate a report for the month with the amount of expense per day.

### DISCUSSION THREAD

**Discussion by nloshka404** [firefly-iii/firefly-iii#7909] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)
- [X] My request is not listed as [a very good idea, but unfortunately...](https://docs.firefly-iii.org/firefly-iii/more-information/what-its-not/)
- [X] I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has not been requested before.

### Description

Debug information generated at 2023-08-28 15:21:45 Europe/Moscow for Firefly III version **6.0.18**.

| Scope | Version |
| --- | --- |
| Firefly III | 6.0.18 |
| Firefly III API | 2.0.4 |
| DB version | 19 (exp. 19) |
| Docker | No |
| PHP | `8.2.9` |
| Host | `Linux` |

| System info | Value |
| --- | --- |
| System TZ | Europe/Moscow |
| Browser TZ | Europe/Moscow |
| App environment | local |
| App debug mode | false |
| App cache driver | file |
| App logging | notice, daily |
| Display errors | Off |
| Error reporting | ALL errors |
| Max upload | 2097152 (2 MB) |
| Interface | fpm-fcgi |
| Default language | ru_RU |
| Default locale | equal |
| BCscale | 12 |
| DB drivers | mysql, sqlite |
| Current driver | mysql |
| Trusted proxies (.env) | `` |

| User info | Value |
| --- | --- |
| Login provider | eloquent  |
| User guard | web  |
| Headers | N/A, N/A  |
| Stateful domains |  |
| Session start | 2023-07-01 15:21:35 |
| Session end | 2023-07-31 15:21:35 |
| Session first | 2019-01-01 00:00:00 |
| User ID | 1 |
| User language | ru_RU |
| User locale | ru_RU |
| Attempt at "ru_RU.utf8" | 'ru_RU.utf8' |
| Attempt at "ru_RU.UTF-8" | 'ru_RU.UTF-8' |
| User agent | Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 |

### Solution

good time of the day. Tell me how you can generate a report for the month with the amount of expense per day.
What would every day display the expense for the day and in total for the month. You need to find a mistake, and it is difficult to look for a mistake for a whole month. It is easier to find an error in days.
Is it possible to withdraw transaction amounts per day?
If not, I suggest doing it in future versions.

**Comment by JC5** [comment]
Are you looking for the audit report perhaps?

**Reply by nloshka404** [reply]
maybe you need the program to represent the amount of expenses per day as it represents the amount of expenses by months in the account section.
I need to find the filling error, I know for sure that the error is in one month, but in order to find the error, you need to check the entire month. it's about 5000 records. It’s easier to check for a day in some days there is an error for sure, but in which ones you need to find if there was a presentation of amounts per day, you could easily find a filling error for a specific day. It is very long and difficult to search for an error in a month, but it is possible. Is it possible to facilitate the search for errors by making a daily representation of the costs? After all, the representation of how much is spent and earned on a monthly basis is there a daily one?

**Reply by nloshka404** [reply]
if there was a presentation of expenses per day, then it would be easy to find an error simply by comparing the amounts of expenses and income per day. and it would be possible to see in which day the error is. Since there is already a printout from the bank account, which shows the amount of expenses and income per day. It would be easier to compare the amounts for the day and the error would be immediately visible. than looking for an error in a whole month, which is much more difficult to do.

**Reply by JC5** [reply]
Sorry, but for 5000 records per month? Are you using this for your business? This is out of my league I'm afraid.

**Reply by nloshka404** [reply]
I don’t use it for business, these are the expenses of records, well, for example, some trifle was bought for a small amount and about 5000 records can be collected per month, even coffee at a gas station bought for 1 ruble also needs to be entered into the program so that it correctly displays the balance and takes into account consumption.

**Reply by nloshka404** [reply]
As a result, it takes a long time to look for an error
Made it easier: Made a reconciliation and entered a corrective amount as a result.
This evened out the final balance figure.
If an error is accidentally discovered, then I will correct it, but intentionally looking for an error, there is no desire to especially look at old records, it is rare, but sometimes it is interesting what and where was spent.
